api/pkg/s3/bucketload.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s3
```

Comment:
please share link to design doc for this feature, thanks.
Specifically, 
- if these objects are used for analytics, where are they stored during the analysis ?
- if we are going to store objects from multiple storage types, in our DB, how will we design to distinguish between them ?

Reply:
Please go to the top of this PR and check the release note, thanks.